### PR TITLE
WFP-1017 added person manager allocated event definition

### PR DESCRIPTION
**Pull request by carlov20**
`{}` are all place holders for values which will be passed in at run time

### CODE REVIEW

File: personManagerAllocatedEventDefinition.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "eventType": "PERSON_MANAGER_ALLOCATED",
```
Comment (carlov20) [inline personManagerAllocatedEventDefinition.json:2]: I noticed in the ADR event type was lower case and `.` separated, does this need to be changed to reflect that? is that the preferred naming strategy for event types?

Comment (marcus-bcl) [reply personManagerAllocatedEventDefinition.json:2]: I think so yes - the probation data events were originally created in this format (uppercase with underscores), however this was re-defined for HMPPS domain events.  @mxco86 might be able to provide a link to the message spec in Confluence.

Comment (carlov20) [reply personManagerAllocatedEventDefinition.json:2]:
```suggestion
  "eventType": "person.manager.allocated",
```
as a first pass for an alternative

**Review by markberridge — APPROVED** [review]
It matches the [ADR](https://dsdmoj.atlassian.net/wiki/spaces/NDSS/pages/3732570166/ADR+-+Domain+Event+Schema) we just need to move that from a "proposed" to "approved" state to help ensure consistency going forward

**Pull request merged**